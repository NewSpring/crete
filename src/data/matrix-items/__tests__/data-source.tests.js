import MatrixItemDS from '../data-source';

describe('the matrix item data source', () => {
  let MatrixItem;
  beforeEach(() => {
    MatrixItem = new MatrixItemDS();
  });
  it('returns an empty list for a blank guid', async () => {
    expect(await MatrixItem.getItemsFromGuid('')).toMatchSnapshot();
  });
});
