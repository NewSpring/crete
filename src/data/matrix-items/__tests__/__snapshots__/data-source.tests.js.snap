// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the matrix item data source returns an empty list for a blank guid 1`] = `Array []`;
